packages/kbn-handlebars/index.ts

```diff
@@ -590,77 +612,51 @@ class ElasticHandlebarsVisitor extends Handlebars.Visitor {
   }
 
   private processAmbiguousNode(node: ProcessableNodeWithPathParts) {
-    const invokeResult = this.invokeAmbiguous(node);
-
-    if (isBlock(node)) {
-      const result = this.ambiguousBlockValue(node, invokeResult);
-      if (result != null) {
-        this.output.push(result);
-      }
-    } else {
-      if (
-        (node as hbs.AST.MustacheStatement).escaped === false ||
-        this.compileOptions.noEscape === true ||
-        typeof invokeResult !== 'string'
-      ) {
-        this.output.push(invokeResult);
-      } else {
-        this.output.push(Handlebars.escapeExpression(invokeResult));
-      }
-    }
-  }
-
-  // This operation is used when an expression like `{{foo}}`
-  // is provided, but we don't know at compile-time whether it
-  // is a helper or a path.
-  //
-  // This operation emits more code than the other options,
-  // and can be avoided by passing the `knownHelpers` and
-  // `knownHelpersOnly` flags at compile-time.
```

Comment:
__question:__ is this comment no longer applicable?

Reply:
It was copied from the upstream handlbears project ([source](https://github.com/handlebars-lang/handlebars.js/blob/e6686963ba1140ad6d8d10f7f302d1adf6b8740f/lib/handlebars/compiler/javascript-compiler.js#L718-L729)). But in this commit I refactor a bunch of the logic so the function it was attached to is no longer there, but merged with two other functions. So I either had to reword it or drop it. I choose the latter as I didn't feel it was needed